### ISSUE TITLE
Request for Loss Cross Entropy Weights for HerdNet General Dataset

### ISSUE THREAD

**Issue by idchacon28** [Alexandre-Delplanque/HerdNet#12]
Hello Alexandre,  
  
I've been working with your repository and I appreciate the set of weights provided for the Ennedi dataset (`weights=[0.1,5.,15.,1.]`) and the work you have done here!  
  
However, I'm currently interested in trying to reproduce your training results on the HerdNet General dataset, which contains a larger number of classes. I noticed that the `.yaml` configuration file does not include the loss cross entropy weights for this particular dataset.  
  
Would it be possible for you to provide the corresponding weights for the HerdNet General dataset? Having these weights would be immensely beneficial for accurately replicating your results.  
  
Thank you for your time and assistance.  
  
Best regards,  
Daniel C

**Comment by idchacon28** [comment]
Oh, I think I actually found them in your demo notebook:
weight = Tensor([0.1, 1.0, 2.0, 1.0, 6.0, 12.0, 1.0])
